its/ruling/src/test/resources/expected/kotlin/kotlin-language-server/kotlin-S5612.json

```diff
@@ -0,0 +1,15 @@
+{
+'kotlin-kotlin-language-server-project:server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt':[
+69,
```

Comment:
I created [SONARKT-249](https://sonarsource.atlassian.net/browse/SONARKT-249) to fix this false-positive.